CHANGELOG.md

```diff
@@ -6,6 +6,9 @@
 * This library requires Turf v2.0.0-alpha.3. ([#525](https://github.com/mapbox/mapbox-directions-swift/pull/525))
 * The `Incident.impact` property is now an `Incident.Impact` value instead of a string. ([#519](https://github.com/mapbox/mapbox-directions-swift/pull/519))
 
+## v2.0.0
+* Added the `Intersection.preferredApproachLanes`, `Intersection.usableLaneIndication`, `Lane.isActive`, and `Lane.validIndication` properties that indicate preferred lane usage. ([#529](https://github.com/mapbox/mapbox-directions-swift/pull/529))
```

Comment:
The `Lane` struct is internal, so we shouldn’t mention it in the changelog; the additions to `Intersection` are what concern developers. However, we should also mention that the `VisualInstruction.Component.lane(indications:isUsable:)` case has been renamed to `VisualInstruction.Component.lane(indications:isUsable:preferredDirection:)`.